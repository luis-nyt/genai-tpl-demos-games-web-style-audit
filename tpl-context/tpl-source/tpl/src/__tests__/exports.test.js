import { join } from 'path';
import getExports from '../../../foundation/web/src/__tests__/getExports.js';

const path = join(__dirname, '../index.ts');

const tplExports = getExports(path).sort();

it('contains the expected named exports', () => {
  expect(tplExports).toMatchSnapshot();
});
