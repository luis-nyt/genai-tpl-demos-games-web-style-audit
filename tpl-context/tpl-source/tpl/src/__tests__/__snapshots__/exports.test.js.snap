// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`contains the expected named exports 1`] = `
Array [
  "AlertIcon",
  "AppleIcon",
  "ArrowDownIcon",
  "ArrowLeftIcon",
  "ArrowRightIcon",
  "ArrowUpIcon",
  "BackgroundColor",
  "BackgroundColorModifier",
  "BaseButton",
  "BaseButtonAsButtonProps",
  "BaseButtonAsLinkProps",
  "BaseButtonCustomProps",
  "BaseButtonHeight",
  "BaseButtonOverload",
  "BaseButtonProps",
  "Body",
  "BodyProps",
  "BorderRadius",
  "BorderWidth",
  "Box",
  "BoxProps",
  "BreakpointArray",
  "BreakpointContext",
  "Button",
  "ButtonProps",
  "ButtonStyledProps",
  "ButtonWeight",
  "CSSPropertiesAndCustomProperties",
  "CalendarIcon",
  "CaretDownIcon",
  "CaretLeftIcon",
  "CaretRightIcon",
  "CaretUpIcon",
  "CheckCircleIcon",
  "CheckmarkIcon",
  "CloseIcon",
  "ColorBehaviorContext",
  "ColorClassParams",
  "CommonProps",
  "CommonWithAsProps",
  "CookingIcon",
  "CreateFocusClassOptions",
  "CreateIcon",
  "DATA_DARK_MODE",
  "DataTplValue",
  "Dialog",
  "DialogContent",
  "DialogContentProps",
  "DialogProps",
  "DialogStyle",
  "DialogStyleCssInJsProperties",
  "DotIcon",
  "FacebookIcon",
  "Flex",
  "FlexProps",
  "FolderIcon",
  "GamesIcon",
  "GoogleIcon",
  "HRule",
  "HRuleProps",
  "Headline",
  "HeadlineFeature",
  "HeadlineFeatureProps",
  "HeadlineNews",
  "HeadlineNewsProps",
  "HeadlineOpinion",
  "HeadlineOpinionProps",
  "HeadlineProps",
  "HelpIcon",
  "HistoryIcon",
  "HomeIcon",
  "Icon",
  "IconButton",
  "IconButtonProps",
  "IconButtonStyledProps",
  "IconComponentProps",
  "IconProps",
  "IconSize",
  "InfoIcon",
  "Label",
  "LabelProps",
  "Link",
  "LinkBox",
  "LinkBoxProps",
  "LinkIcon",
  "LinkProps",
  "MenuIcon",
  "MoreHorizIcon",
  "MoreVertIcon",
  "MutedIcon",
  "NewspaperIcon",
  "PasswordInput",
  "PasswordInputProps",
  "PauseIcon",
  "PersonIcon",
  "PlayCircleIcon",
  "PlayIcon",
  "PlusIcon",
  "Radio",
  "RadioGroup",
  "RadioGroupProps",
  "RadioProps",
  "RefreshIcon",
  "ResponsiveTypography",
  "Root",
  "RootProps",
  "Row",
  "RowDescription",
  "RowDescriptionProps",
  "RowGroup",
  "RowGroupProps",
  "RowHeader",
  "RowHeaderProps",
  "RowLabel",
  "RowLabelProps",
  "RowProps",
  "SaveFilledIcon",
  "SaveIcon",
  "SearchIcon",
  "SettingsIcon",
  "ShareAndroidIcon",
  "ShareAppleIcon",
  "ShareIcon",
  "SpaceProps",
  "SpaceType",
  "Spinner",
  "SpinnerProps",
  "StarIcon",
  "StoryList",
  "StoryListHeader",
  "StoryListHeaderProps",
  "StoryListItem",
  "StoryListItemContent",
  "StoryListItemContentProps",
  "StoryListItemProps",
  "StoryListMediaPosition",
  "StoryListMediaPositionContext",
  "StoryListOuterPaddingOptions",
  "StoryListProps",
  "StoryListSection",
  "StoryListSectionProps",
  "StoryListStyle",
  "StoryListStyleCssInJsProperties",
  "StrokeColor",
  "SuperTIcon",
  "Switcher",
  "SwitcherProps",
  "Text",
  "TextButton",
  "TextButtonProps",
  "TextButtonStyledProps",
  "TextColor",
  "TextInput",
  "TextInputProps",
  "TextProps",
  "TheAthleticIcon",
  "Theme",
  "Title",
  "TitleKarnak",
  "TitleKarnakProps",
  "TitleProps",
  "Toast",
  "ToastProps",
  "ToastStyle",
  "ToastStyleCssInJsProperties",
  "TplSx",
  "TypeStyle",
  "Typography",
  "TypographyKey",
  "TypographyPrefix",
  "TypographyProps",
  "TypographyVariant",
  "UnmutedIcon",
  "VRule",
  "VRuleProps",
  "WirecutterIcon",
  "baseFocusStyles",
  "borderRadius",
  "borderWidth",
  "breakpoints",
  "color",
  "colorClass",
  "createDialogStyleClass",
  "createDialogStylePropertyDeclarations",
  "createFocusClass",
  "createStoryListStyleClass",
  "createStoryListStylePropertyDeclarations",
  "createToastStyleClass",
  "createToastStylePropertyDeclarations",
  "createTplColorBehaviorStyles",
  "dark",
  "darkIconArtworkClass",
  "dataTplAttr",
  "dialogStyle",
  "disabledSelector",
  "focusNotFocusVisibleSelector",
  "focusSelector",
  "focusStyles",
  "focusVisibleSelector",
  "fontSize",
  "fontWeight",
  "getSpace",
  "horizontalRuleVariants",
  "iconSizes",
  "invertTplColors",
  "light",
  "lightIconArtworkClass",
  "minWidths",
  "notDisabledSelector",
  "prefersDarkMode",
  "rootFocusClass",
  "ruleTokens",
  "spaceScale",
  "storyListOuterPadding",
  "storyListStyle",
  "textBaseClass",
  "toastStyle",
  "typography",
  "useBreakpointContext",
  "useColorBehaviorContext",
  "useResponsivePropToCSS",
  "useResponsivePropToClass",
  "useTplSxProp",
  "verticalRuleVariants",
  "visuallyHidden",
  "visuallyHiddenStyles",
]
`;
